app/services/db/sp_cost/sp_cost_summary_by_iaa.rb

```diff
@@ -0,0 +1,41 @@
+module Db
+  module SpCost
+    class SpCostSummaryByIaa
+      # @param [String] iaa
+      # @param [Range<Date>] iaa_range
+      def self.call(iaa:, iaa_range:)
+        params = {
+          iaa: iaa,
+          iaa_start_date: iaa_range.begin,
+          iaa_end_date: iaa_range.end,
+        }.transform_values { |v| ActiveRecord::Base.connection.quote(v) }
+
+        sql = format(<<~SQL, params)
+          SELECT
+            sp_costs.issuer
+          , sp_costs.ial
+          , sp_costs.cost_type
+          , MAX(service_providers.iaa) AS iaa
+          , MAX(service_providers.app_id) AS app_id
```

Comment:
there can be multiple app_id's per iaa so you can drop app_id from this query.

Reply:
Thanks, will do